maquetado/scss/styles.scss

```diff
@@ -1,32 +1,56 @@
 .bold {
-    font-weight: bold;
+  font-weight: bold;
+}
+
+.book-container {
+  display: flex;
+  flex-direction: row;
+  justify-content: flex-start;
+}
+
+.book-cover {
+  width: 261px;
+  height: 368px;
+}
+
+.column {
+  display: flex;
+  flex-direction: column;
+  width: 400px;
+  margin-left: 100px;
```

Comment:
margin-right in the image. Also, I think it's too big, it looks smaller in the design, don't you think?

Reply:
Hmm, at least the values that are @ invision match with these. It did look big but due to the fact that the text was kinda small.